test/e2e_node/e2e_node_suite_test.go

```diff
@@ -91,7 +91,7 @@ var _ = BeforeSuite(func() {
 	if *startServices {
 		e2es = newE2eService(*nodeName)
 		if err := e2es.start(); err != nil {
-			Fail(fmt.Sprintf("Unable to start node services.\n%v", err))
+			glog.Fatalf(fmt.Sprintf("Unable to start node services.\n%v", err))
```

Comment:
@vishh Initially I did not change it but later thought that it should be fatal if services are not started so added this.  But now I think again that with this, it wont do the work being done in AfterSuite. So I think I will change to the original (Fail) here and will update it soon.